pyproject.toml

```diff
@@ -1,6 +1,55 @@
+[project]
+name = "wagtail-flags"
+version = "5.3.0"
+description = "Feature flags for Wagtail sites"
+readme = "README.md"
+requires-python = ">=3.8"
+license = {text = "CC0"}
+authors = [
+    {name = "CFPB", email = "tech@cfpb.gov" }
+]
+dependencies = [
+    "wagtail>2.15,<4",
+    "django-flags>4.2,<5.1"
```

Comment:
One thing I want to look into before I merge this, I don't recall why I'm restricting this to Django-Flags < 5.1.

Reply:
The latest django-flags version is 5.0.12 -- maybe you didn't trust future you not to release breaking changes in 5.1?

Reply:
Yeah. I'm going to remove that. It's a slight anti-pattern in Python packaging, as much as it disquiets me.